client/benches/torii.rs

```diff
@@ -59,16 +58,27 @@ fn query_requests(criterion: &mut Criterion) {
         IdBox::AssetId(AssetId::new(asset_definition_id, account_id.clone())),
     );
     let mut client_config = iroha_client::samples::get_client_config(&get_key_pair());
+
+    // TODO: We should definitely turn this into type-checked state. `String` is a terrible bodge
```

Comment:
would you mid opening an issue for this?

Reply:
#1786